components/sup/src/manager/service/config.rs

```diff
@@ -82,7 +81,7 @@ impl ServiceConfig {
     pub fn new(package: &PackageInstall,
                runtime: &RuntimeConfig,
                config_root: Option<PathBuf>,
-               bindings: &[String])
+               bindings: Vec<(String, ServiceGroup)>)
```

Comment:
Given the structs we have (like the PackageInstall, RuntimeConfig, might bindings benefit from a newtype to wrap the Vec<(String, ServiceGroup)>?